datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterStatusResponse.java

```diff
@@ -46,5 +46,7 @@
     DATALAKE_BACKUP_INPROGRESS,
     DATALAKE_RESTORE_INPROGRESS,
     DATALAKE_RESTORE_FAILED,
-    DATALAKE_DETACHED
+    DATALAKE_DETACHED,
+    DATALAKE_UPGRADE_CCM_IN_PROGRESS,
+    DATALAKE_UPGRADE_CCM_FAILED
```

Comment:
Just a sanity check: we do not need a `DATALAKE_UPGRADE_CCM_FINISHED` status because it will be ultimately set to `RUNNING`?

Reply:
yes